File: Expenses.UI/ClientApp/src/app/components/expenses/models/expenses-summary.ts
import { ExpenseClaim } from './expense-claim';

export class ExpensesSummary {

  public claims: Array<ExpenseClaim>;
  public totalPaid: number;
  public totalClaimed: number;
  public currency: string;

  constructor(item: ExpensesSummary = {} as ExpensesSummary) {

    const {
      totalClaimed = 0,
      totalPaid = 0,
      currency = '',
      claims = new Array<ExpenseClaim>()
    } = item;

    this.totalClaimed = totalClaimed;
    this.totalPaid = totalPaid;
    this.claims = claims.map(c => new ExpenseClaim(c));
    this.currency = currency;
  }

}

File: expenses/Expenses.API/Models/ExpensesSummary.cs
using System.Collections.Generic;

namespace Expenses.API.Models
{
    public class ExpensesSummary
    {
        public string Currency { get; set; }
        public decimal TotalClaimed { get; set; }
        public decimal TotalPaid { get; set; }
        public List<ExpenseClaim> Claims { get; set; }
    }
}